todo-app/src/todoItems.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+import TodoDescription from './todoDescription'
+import TodoState from './todoState'
+
+class todo extends Component {
```

Comment:
In this project we should follow the convention that component class names start with capital and the name is exactly the same as the file name. So in this case, the class name should `TodoItem` and the filename should be `TodoItem.js`.

You should do that for all your components!